arkouda/strings.py

```diff
@@ -525,6 +525,50 @@ def is_title(self) -> pdarray:
             generic_msg(cmd="checkChars", args=f"isTitle {self.objtype} {self.entry.name}")
         )
 
+    @typechecked
+    def strip(self, pattern: Union[bytes, str_scalars] = " ") -> Strings:
```

Comment:
The parameters should be listed in the docstring. We need to note that pattern is treated as a set of chars and not as a string.

Reply:
This is a bit of a nitpick, but i feel like `pattern` should be renamed to `chars` since that is the name of the parameter for both the python and chapel version of `strip`